### ISSUE TITLE
Add @simplewebauthn/typescript-types to dependencies of @simplewebauthn/server

### ISSUE THREAD

**Issue by dkonasov** [MasterKale/SimpleWebAuthn#368]
## Describe the issue

Because of type package is not listed in regular dependencies of server package, I have some problems with type inference in VSCode, as the compiler tries to get types from `@simplewebauthn/typescript-types`, which is not installed, as it listed only in devDependencies, and not in dependencies.

## Reproduction Steps

1. Install `@simplewebauthn/server` via npm
1. Call `generateAuthenticationOptions`, assign result to variable
1. Try to access any field of result, and get advantage of type hinting in VSCode. There will be no hint, also there will be no type checking for the argument of `generateAuthenticationOptions`

## Expected behavior

Type hints are displayed in the IDE

## Code Samples + WebAuthn Options and Responses

https://github.com/dkonasov/simplewebauthn-types-demo/blob/main/index.ts#L5

## Dependencies

- **OS:** macOS 12.0.1
- **Node/NPM** 18.12.0/8.19.2
- **Typescript** 4.9.5

### SimpleWebAuthn Libraries

```sh
$ npm list --depth=0 | grep @simplewebauthn
@simplewebauthn/server@7.0.1
# ...
```

**Comment by MasterKale** [comment]
@dkonasov Thank you for reporting this, I'm realizing now that I should have had `typescript-types` included in `"dependencies"` this whole time. I'll go ahead and update that so everyone always gets the types too.

**Comment by MasterKale** [comment]
@dkonasov I just published **@simplewebauthn/server@7.2.0** (and **@simplewebauthn/browser@7.2.0**) that should resolve this issue 🚀
